src/lucky/page_helpers/text_helpers.cr

```diff
@@ -126,6 +126,54 @@ module Lucky::TextHelpers
     end
   end
 
+  # Creates a comma-separated sentence from the provided `Enumerable` *list*
+  # and appends it to the view.
+  #
+  # #### Options:
+  #
+  # The following options allow you to specify how the sentence is constructed:
+  #   - *word_connector* - A string used to join the elements in *list*s
+  # containing three or more elements (Default is ", ")
+  #   - *two_word_connector* - A string used to join the elements in *list*s
+  # containing exactly two elements (Default is " and ")
+  #   - *last_word_connector* - A string used to join the last element in
+  # *list*s containing three or more elements (Default is ", and ")
+  #
+  # #### Examples:
+  #
+  #     to_sentence([] of String)            # => ""
+  #     to_sentence([1])                     # => "1"
+  #     to_sentence(["one", "two"])          # => "one and two"
+  #     to_sentence({"one", "two", "three"}) # => "one, two, and three"
+  #
+  #     to_sentence(["one", "two", "three"], word_connector: " + ")
+  #     # => one + two, and three
+  #
+  #     to_sentence(Set{"a", "z"}, two_word_connector: " to ")
+  #     # => a to z
+  #
+  #     to_sentence(1..3, last_word_connector: ", or ")
+  #     # => 1, 2, or 3
+  #
+  # NOTE: By default `#to_sentence` will include a
+  # [serial comma](https://en.wikipedia.org/wiki/Serial_comma). This can be
+  # overriden like so:
```

Comment:
I like the trivia about the oxford comma :) Not sure it warrants an extra `NOTE`, but also happy to see it here. `overriden => overridden`.

Reply:
Ah! Thanks for the catch of the misspelling! Sorry that it snuck through!

As far as the note is concerned, I noticed there was a [comment on the Rails documentation](https://apidock.com/rails/Array/to_sentence#1497-Grammatical-error) expressing some concern about the Oxford comma, so I figured it might be helpful to mention it. Do you think this is worth including? 😄 If so, do you think it should it be under a `NOTE` (I saw that `NOTE`s get handled specially by `crystal docs`)?

Reply:
No worries about the typos, not much of a problem to have one sneak through :) I think the information per se is worth including, it makes the docs a bit more lively and interesting than just plain stating method documentation / arguments etc. Just my 2 cents :) Looking at the crystal api docs, the notes are more technically relevant I guess, so we should probably not overuse them, but 🤷‍♂️ I like it the more I think about it haha

Reply:
I like it a lot! It's a nice thing to know and it is helpful to know how to override it when necessary